src/arrayMethodJoin.js

```diff
@@ -4,8 +4,22 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let newString = '';
+
+    for (let i = 0; i < this.length; i++) {
+      const element = this[i];
+
+      if (element !== undefined && element !== null) {
+        newString += `${element}`;
```

Comment:
it will work this way too:
```suggestion
        newString += element;
```